### PR TITLE
BaSP-TG-09: Admins table

### PR DESCRIPTION
**Pull request by amilcarHernandez24**

### CODE REVIEW

**Comment by TomasBettini** [comment]
The name of your PR is wrong, your assigned ticket is 09 and I don't know why but the deploy to vercel has a problem

File: src/Components/Admins/FormAdmins/index.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  }
+
+  const createAdmin = async () => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}admins`, {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify(adminInput)
+      });
+      console.log(adminInput);
```
Comment (TomasBettini) [inline src/Components/Admins/FormAdmins/index.js:46]: You must delete this console.log

Comment (amilcarHernandez24) [reply src/Components/Admins/FormAdmins/index.js:46]: thanks, I'll

File: src/Components/Admins/index.js [hunk 2]
```diff
@@ -1,11 +1,37 @@
...
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}admins`);
+      const data = await response.json();
+      setAdmins(data);
+    } catch (error) {
+      console.error(error);
+    }
+  }, [admins]);
+
+  // eslint-disable-next-line no-unused-vars
```
Comment (TomasBettini) [inline src/Components/Admins/index.js:18]: You must delete this comment

**Review by TomasBettini — APPROVED** [review]
I think you still have to add a modal to show the success and error messages, and also add a bit of css to the form and the list, but I see that the functionalities are all good

File: src/Components/Admins/index.js (hunk 2 above)
Comment (amilcarHernandez24) [reply src/Components/Admins/index.js:18]: Thanks, I'll delete it.

**Review by juliancit0 — CHANGES_REQUESTED** [review]

File: src/Components/Admins/index.js [hunk 3]
```diff
@@ -1,11 +1,36 @@
...
+    }
+  }, [admins]);
+
+  const deleteAdmin = (id) => {
+    try {
+      fetch(`${process.env.REACT_APP_API_URL}/admins/${id}`, { method: 'DELETE' });
+    } catch (error) {
+      console.error(error);
+    }
+  };
```
[the remark this answers is not in the document]
Comment (amilcarHernandez24) [reply src/Components/Admins/index.js:24]: Thanks Juli. I'll change that

**Review by juliancit0 — DISMISSED** [review]

File: src/Components/Admins/index.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (juliancit0) [reply src/Components/Admins/index.js]: Nice!

File: src/Components/Admins/index.js [hunk 4]
```diff
@@ -1,11 +1,43 @@
...
+      console.error(error);
+    }
+  }, []);
+
+  const deleteAdmin = async (id) => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/admins/${id}`, {
+        method: 'DELETE'
+      });
+      if (response.ok) {
```
Comment (mgriolfo-g) [inline src/Components/Admins/index.js:23]: A filter for a deleted element should be added after the request of its deletion (or when its deletion is confirmed), in order to keep the page dynamic and avoiding errors. Try to add a filter like this one:
`        const newAdmins = admins.filter((admin) => admin._id !== id);
        setAdmins(newAdmins);`

Comment (amilcarHernandez24) [reply src/Components/Admins/index.js]: I just fixed it

**Review by leimeter-joaquin — APPROVED** [review]

**Review by TomasBettini — APPROVED** [review]
Nice job!

**Review by leimeter-joaquin — APPROVED** [review]

**Pull request merged**